ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml

```diff
@@ -101,6 +101,11 @@ tests:
     env:
       CLUSTER_TYPE_OVERRIDE: openstack-vh-mecha-az0
     workflow: shiftstack-monitor
+- as: aliveness-mecha-central
```

Comment:
Nit, industry seems to favor the `liveliness` term for this type of check.

Reply:
oh cool, I'll change it